app/views/feed/index.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= render "layouts/account", title: t("account.navigation.feed") do %>
+  <% if current_user.favorites.any? %>
+    <h2 class="feed__heading">New updates</h2>
+
+    <% if @feed_last_visited_at && @revisions.first.created_at < @feed_last_visited_at %>
+      <p class="feed__message">There are no new updates since your last visit, which was <%= time_ago_in_words(@feed_last_visited_at) %> ago.</p>
```

Comment:
```suggestion
      <p class="feed__message">No new updates since your last visit, which was <%= time_ago_in_words(@feed_last_visited_at) %> ago.</p>
```
for conciseness